CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ Merged [Pull Request](https://github.com/MethodsAndPractices/vsteam/pull/502) fr
 - Updated test c# project to .NET 6.0
 - Fixed test fail on ubuntu-latest
 
+Merged [Pull Request](https://github.com/MethodsAndPractices/vsteam/pull/506) from [Miguel Nieto](https://github.com/mnieto) the following:
+- fix tests that involve dates fail when OS culture has 24 hours format
+
```

Comment:
Can you update to the latest (not released) version 7.12.0?

Reply:
updated